docs/podman-pod-inspect.1.md

```diff
@@ -0,0 +1,48 @@
+% podman-pod-inspect "1"
+
+## NAME
+podman\-pod\-inspect- Displays information describing a pod
```

Comment:
nit: space in between inspect and -
s/inspect-/inspect -

Reply:
fixed